### PR TITLE
fix template generated doc header

### PR DESCRIPTION
**Pull request by ScottSuarez**
**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

File: tpgtools/templates/resource.html.markdown.tmpl [hunk 1]
```diff
@@ -10,23 +10,22 @@
...
-//     and is based on the DCL (https://github.com/GoogleCloudPlatform/declarative-resource-client-library).
-//     Changes will need to be made to the DCL or Magic Modules instead of here.
-//
-//     We are not currently able to accept contributions to this file. If changes
-//     are required, please file an issue at https://github.com/hashicorp/terraform-provider-google/issues/new/choose
-//
-// ----------------------------------------------------------------------------
-
+	limitations under the License. */ -}}
 ---
```
Comment (slevenick) [inline tpgtools/templates/resource.html.markdown.tmpl:14]: Does this need to be above the comment?

**Review by slevenick — APPROVED** [review]
Thanks! This is breaking the website display

File: tpgtools/templates/resource.html.markdown.tmpl (hunk 1 above)
Comment (ScottSuarez) [reply tpgtools/templates/resource.html.markdown.tmpl:14]: the `-}}` prevents it from making a newline

Comment (ScottSuarez) [reply tpgtools/templates/resource.html.markdown.tmpl:14]: consistent with mmv1

**Comment by ScottSuarez** [comment]
notice - @rileykarson

**Comment by rileykarson** [comment]
Whoops- I completely missed that it was `#` and not `//` in https://github.com/GoogleCloudPlatform/magic-modules/pull/5016

**Pull request merged**